<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-BGGDLR48" failures="1" tests="5" name="com.testing_features.Iretry_Analyser_Concept" time="0.036" errors="0" timestamp="2022-01-25T21:28:54 IST" skipped="3">
  <testcase classname="com.testing_features.Iretry_Analyser_Concept" name="login2" time="0.004">
    <skipped/>
  </testcase> <!-- login2 -->
  <system-out/>
  <testcase classname="com.testing_features.Iretry_Analyser_Concept" name="login" time="0.019"/>
  <system-out/>
  <testcase classname="com.testing_features.Iretry_Analyser_Concept" name="login2" time="0.006">
    <failure message="expected:&lt;b[]as&gt; but was:&lt;b[h]as&gt;" type="junit.framework.ComparisonFailure">
      <![CDATA[junit.framework.ComparisonFailure: expected:<b[]as> but was:<b[h]as>
at junit.framework.Assert.assertEquals(Assert.java:100)
at junit.framework.Assert.assertEquals(Assert.java:107)
at com.testing_features.Iretry_Analyser_Concept.login2(Iretry_Analyser_Concept.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.retryFailed(TestInvoker.java:213)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:58)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- login2 -->
  <system-out/>
  <testcase classname="com.testing_features.Iretry_Analyser_Concept" name="login2" time="0.002">
    <skipped/>
  </testcase> <!-- login2 -->
  <system-out/>
  <testcase classname="com.testing_features.Iretry_Analyser_Concept" name="login2" time="0.005">
    <skipped/>
  </testcase> <!-- login2 -->
  <system-out/>
</testsuite> <!-- com.testing_features.Iretry_Analyser_Concept -->
